fuzz/storage-proof/README.md

```diff
@@ -6,7 +6,10 @@ Install dependencies:
 ```
 $ sudo apt install build-essential binutils-dev libunwind-dev
 ```
-
+or on nix:
```

Comment:
I see what you did here :)

Reply:
(I lost some time trying to install binutils, libunwind-dev etc. on nix not realising that I could just install honggfuzz.)